### PR TITLE
Adds the first iteration of HTML examples

### PR DESCRIPTION
**Pull request by schalkneethling**
This is the first alpha version to get HTML examples into the build pipeline. After @kenrick95 latest PR merges, I need to update this and will then open a PR for the tabbed UI for HTML examples.

### CODE REVIEW

File: tmpl/live-html-tmpl.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    %example-css-src%
+  </head>
+  <body>
+      <header class="output-header hidden">
+        <h4>%title%</h4>
+        <button id="reset" class="reset" type="button">Reset</button>
+        <button id="run" class="run" type="button">Run</button>
+      </header>
+      <section class="editor-container">
+<div id="html-editor" class="editor-wrapper"><pre><code>%example-code%</pre></code></div>
```
Comment (schalkneethling) [inline tmpl/live-html-tmpl.html]: Indent intentional

**Comment by schalkneethling** [comment]
@wbamberg This ok to merge?

**Comment by wbamberg** [comment]
@schalkneethling, sorry to be slow looking at this PR. I'll look at it today. I wasn't sure if you were going to resolve the merge conflicts before reviewing.

**Comment by schalkneethling** [comment]
> I wasn't sure if you were going to resolve the merge conflicts before reviewing.

Also, this is not final in any way, shape or form. It just shows it working with the current build process. More work is waiting in the wings once this has been merged.

I wanted to break things up this way to ensure I do not hurt the contributor work flow.

**Comment by wbamberg** [comment]
As I understand it, this does a few things:

* adds dependencies on the CodeMirror modes for displaying HTML
* refactors index.js into a collection of smaller modules
* adds the HTML editor interface

I've not done a detailed review of the code and am not sure I'm qualified to, but as far as I can tell this looks and works great. I tried adding a CSS file to the bundle and that worked fine too.

IMO we should show the output by default, and not require the user to press "Run". Unlike JS, but like CSS, these things tend to be highly visual, and it's helpful for people to be able to see what an element looks like without having to interact with the editor.

We should think about the wording of "Run". It doesn't seem right to me here. But we should test that.

**Comment by schalkneethling** [comment]
> IMO we should show the output by default, and not require the user to press "Run".

Agree

> We should think about the wording of "Run". It doesn't seem right to me here. But we should test that.

Agree here as well. I chose run based on the convention for other interactive code editors like jsFiddle and Codepen. Also thinking ahead for when there are actual tabs for CSS and JS as well.

**Pull request merged**